src/v2/Apps/Fair/Routes/FairOverview.tsx

```diff
@@ -10,7 +12,33 @@ interface FairOverviewProps {
 const FairOverview: React.FC<FairOverviewProps> = ({ fair }) => {
   return (
     <>
-      <FairHeaderFragmentContainer fair={fair} />
+      <FairHeader fair={fair} />
+
+      <CSSGrid
+        gridRowGap={3}
+        gridColumnGap={3}
+        gridTemplateColumns={["repeat(1fr)", "repeat(2, 1fr)"]}
+        mt={3}
+        pt={3}
+        borderTop="1px solid"
+        borderColor="black10"
+      >
+        {fair.articles.edges.length > 0 && (
+          <Box>
+            <Text variant="subtitle" as="h3" mb={2}>
+              Coverage by Artsy Editorial
+            </Text>
+
+            <FairEditorial fair={fair} />
+          </Box>
+        )}
+
+        <Box>
```

Comment:
Given we're de-prioritizing the curated highlights, can you remove this? I believe the editorial list should appear in the center in this case!

(similar to this:)
![image](https://user-images.githubusercontent.com/2081340/90660422-e1df8680-e213-11ea-9578-d0712940ca84.png)

Reply:
👍 

![](http://static.damonzucconi.com/_capture/DGWHxg2bUIsY.png)